core/domain/stats_services.py

```diff
@@ -646,11 +657,15 @@ def delete_playthroughs_multi(playthrough_ids):
     Args:
         playthrough_ids: list(str). List of playthrough IDs to be deleted.
     """
+
+    def _delete_playthroughs_multi():
```

Comment:
Nit, but I think conventionally this would be called ``_delete_playthroughs_multi_transactional`` to explicitly call out that it's meant to be run in a transaction.

Reply:
Done.